README.md

```diff
@@ -80,6 +80,50 @@ case object GetPeople extends CollectionQuery[Seq, Person] {
 val person = db(GetPeople).head // Person(Coda Hale,chale@example.com,29)
 ```
 
+You can also use the sql string interpolator:
```

Comment:
I _can_ use it, but _should_ I? This seems like a pretty major departure from the existing style, so I think some more guidance is needed.

Do we expect this to be the preferred style going forward? If so, present it first and we can call the old style "legacy" or something like that.

If this alternative is appropriate in some situations, but not in others, we should make that clear here.